server/service/src/report/string_or_vec.rs

```diff
@@ -0,0 +1,34 @@
+use std::fmt;
+
+use serde::de::{self, value, Deserialize, Deserializer, SeqAccess, Visitor};
+
+pub(crate) fn string_or_vec<'de, D>(deserializer: D) -> Result<Vec<String>, D::Error>
```

Comment:
Might be nice to have a test for this,  looks pretty safe.

Reply:
Agree, I tested manually but should add one if there will be another change to it!